src/mixer/plugins/volume_mapping.cxx

```diff
@@ -31,8 +31,6 @@
  */
 
 #define _ISOC99_SOURCE /* lrint() */
-#define _GNU_SOURCE /* exp10() */
```

Comment:
I don't get what you mean by amending the original commit. Do you suggest I "reset --hard" back to that commit and re commit without the line?